services/galley/src/Galley/Cassandra/GetAllTeamFeatureConfigs.hs

```diff
@@ -40,6 +40,7 @@ data AllTeamFeatureConfigsRow = AllTeamFeatureConfigsRow
     -- conference calling
     conferenceCalling :: Maybe FeatureStatus,
     conferenceCallingTtl :: Maybe FeatureTTL,
+    conferenceCallingSftForOne2One :: Maybe Bool,
```

Comment:
there is an old rule that we *always* use int instead of bool in cassandra, because it is more extensible.  not sure we'll ever need that here, or how.  just saying :)

Reply:
Switched to `Int`.